src/xGadgetron/pGadgetron/Gadgetron.py

```diff
@@ -1435,14 +1435,21 @@ def set_input(self, input_data):
         '''
         assert isinstance(input_data, AcquisitionData)
         self.input_data = input_data
-    def process(self):
+    def process(self, dcm_prefix=None):
```

Comment:
I'd prefer to have a `set_dcm_prefix` or similar function. Adding it as an optional argument is rather non-standard.

Reply:
done